com.wps.WPS.json

```diff
@@ -0,0 +1,104 @@
+{
+    "app-id": "com.wps.WPS",
+    "branch": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "tags": ["proprietary"],
+    "command": "wps",
+    "separate-locales": false,
+    "build-options": {
+        "no-debuginfo": true
+    },
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=host"
+    ],
+    "cleanup": [
+        "/share/qt4",
+        "/lib"
+    ],
+    "modules": [
+        "shared-modules/qt4/qt4-4.8.7-minimal.json",
+        {
+            "name": "wps-i18n",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/office6",
+                /* remove translations contained upstream release tarball */
+                "rm -frv en_US zh_CN",
+                "make install && mv -v ~/.kingsoft/office6/mui /app/share/office6"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/wps-community/wps_i18n.git",
+                    "commit": "39da55042512a19abb6a73b33c0da626c6e470ce"
+                }
+            ]
+        },
+        {
+            "name": "wps",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 apply_extra /app/bin/apply_extra",
+                "install -Dm755 wpsrename /app/bin/wpsrename",
+                "ln -s ../extra/wps-office/wps /app/bin/wps",
+                "ln -s ../extra/wps-office/et /app/bin/et",
+                "ln -s ../extra/wps-office/wpp /app/bin/wpp",
+                "mkdir -p /app/share/metainfo/",
+                "cp com.wps.WPS*.appdata.xml /app/share/metainfo/"
+            ],
+            "sources": [
+                {
+                    "type": "script",
+                    "dest-filename": "wpsrename",
+                    "commands": [
+                        "mv $1 $(dirname $1)/$(basename $1|sed -e 's/wps-office-/com.wps.WPS-/')"
```

Comment:
You should also use desktop-file-edit to set the old name in X-Flatpak-RenamedFrom. This will allow GNOME Software, Shell, etc or other tools to map between shortcuts or other metadata that apply to the original name. Note this key is a list so it should be terminated with ;.